connector/producer.py

```diff
@@ -45,7 +46,7 @@
 @openerp.api.returns('self', lambda value: value.id)
 def create(self, vals):
     record_id = create_original(self, vals)
-    if self.pool.get('connector.installed') is not None:
+    if is_module_installed(self.env, 'connector') is not None:
```

Comment:
No, I just replace pool by env in the goal to get the cursor, If you prefer I can replace self.env by self